src/global.css

```diff
@@ -0,0 +1,40 @@
+body {
```

Comment:
It's not the first time you make such a mistake 
Please avoid element or any other type of selector except for the class selector